core_data_modules/data_models/scheme.py

```diff
@@ -2,19 +2,23 @@
 
 
 class Scheme(object):
-    scheme_id = None
-    name = None
-    version = None
-    codes = []
-    documentation = dict()
+    def __init__(self, scheme_id, name, version, codes, documentation=None):
+        if documentation is None:
```

Comment:
Why not default it to a dictionary?

Reply:
As discussed yesterday, because then the same dictionary would be shared across all scheme instances.

Reply:
So to be clear  
```
def __init__(self, scheme_id, name, version, codes, documentation={}):
```

Shares the generated dictionary across all instances?

Reply:
Yes (eww)